manifests/pipelines/s3-fetch-pipeline.yaml

```
@@ -314,21 +346,31 @@ spec:
     - test-model-rest-svc
     - stop-deployment
     workspaces:
-    - name: images-url
+    - name: images_url
       workspace: build-workspace-pv
   - name: skopeo-copy
     params:
+    - name: SOURCE_IMAGE_URL
+      value: $(tasks.retrieve-build-image-info.results.source-image-url)
+    - name: DESTINATION_IMAGE_URL
+      value: $(tasks.retrieve-build-image-info.results.dest-image-url)
```

After that, this can be left empty:
```suggestion
    - name: SOURCE_IMAGE_URL
      value: ""
    - name: DESTINATION_IMAGE_URL
      value: ""
```